src/layers/core/hexagon-cell-layer/hexagon-cell-layer-vertex.glsl.js

```diff
@@ -29,11 +29,6 @@ attribute vec3 instancePositions;
 attribute vec4 instanceColors;
 attribute vec3 instancePickingColors;
 
-// Picking uniforms
-// Set to 1.0 if rendering picking buffer, 0.0 if rendering for display
-uniform float renderPickingBuffer;
-uniform vec3 selectedPickingColor;
-
 // Custom uniforms
```

Comment:
@heshan0131 , is this code required for any `Voyager` features? After this PR, `Voyager` can use highlighting features supported by deck.gl ( #839 )